apis/core/v1alpha1/types_shared.go

```diff
@@ -365,3 +365,11 @@ func (r VersionedResourceReference) ObjectMeta() cdv2.ObjectMeta {
 		Version: r.Version,
 	}
 }
+
+type ComponentLibrary string
+
+const (
+	ComponentLibraryCNUDIE  ComponentLibrary = "cnudie"
+	ComponentLibraryOCM     ComponentLibrary = "ocm"
+	componentLibraryDefault ComponentLibrary = ComponentLibraryCNUDIE
+)
```

Comment:
Why are there 3 constants and not two compared to the core types?

Reply:
It defines which of the two is the default.

At some point in the future we wanted to switch the default from `cnudie` to `ocm`. That will be done here, at a single point, by switching from 
```golang
componentLibraryDefault ComponentLibrary = ComponentLibraryCNUDIE
```
to
```golang
componentLibraryDefault ComponentLibrary = ComponentLibraryOCM
```
instead of inspecting all places where the `cnudie` constant occurs.

Reply:
Oh yes, I totally missed this.